static/js/components/ErrorMessage_test.js

```diff
@@ -0,0 +1,214 @@
+/* global SETTINGS: false */
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+import { assert } from 'chai';
+import _ from 'lodash';
+
+import {
+  RECEIVE_DASHBOARD_FAILURE,
+  RECEIVE_GET_USER_PROFILE_SUCCESS,
+  REQUEST_GET_USER_PROFILE,
+  RECEIVE_DASHBOARD_SUCCESS,
+  RECEIVE_GET_USER_PROFILE_FAILURE,
+  SET_USER_PAGE_DIALOG_VISIBILITY,
+  START_PROFILE_EDIT,
+  UPDATE_PROFILE_VALIDATION,
+  REQUEST_PATCH_USER_PROFILE,
+  RECEIVE_PATCH_USER_PROFILE_FAILURE,
+  CLEAR_PROFILE_EDIT,
+} from '../actions';
+import {
+  DASHBOARD_RESPONSE,
+  ERROR_RESPONSE,
+  USER_PROFILE_RESPONSE,
+} from '../constants';
+import IntegrationTestHelper from '../util/integration_test_helper';
+import { makeStrippedHtml } from '../util/util';
+import * as api from '../util/api';
+import ErrorMessage from './ErrorMessage';
+
+describe("ErrorInfo", () => {
+  let errorString = `Sorry, we were unable to load the data necessary
+      to process your request. Please reload the page.`;
+  errorString = errorString.replace(/\s\s+/g, ' ');
+
+  let contactExpectation = `If the error persists, please contact 
+      mitx-support@mit.edu specifying this entire error message.`;
+  contactExpectation = contactExpectation.replace(/\s\s+/g, ' ');
+
+  describe('unit tests', () => {
+    let that;
+    let renderErrorInfo = props => {
+      return makeStrippedHtml(<ErrorMessage {...props} />);
+    };
+    let codeAttributes = [
+      ['error_code', '500'],
+      ['errorStatusCode', 404],
+    ];
+    let messageAttributes = [
+      ['user_message', 'A message'],
+      ['detail', 'Some details'],
+    ];
+
+    beforeEach(() => {
+      that = {
+        errorInfo: {
+          error_code: '500',
+          user_message: 'A message',
+          detail: 'Some details',
+          errorStatusCode: 404,
+        }
+      };
+    });
+
+    it('should take an errorInfo prop', () => {
+      let errorInfo = <ErrorMessage {...that}/>;
```

Comment:
It looks like we're just asserting that the props we passed in were passed in, since there's no render step here.